### ISSUE TITLE
Find kmer locations original sequence

### ISSUE THREAD

**Issue by Adamtaranto** [prophyle/prophex#29]
Hi All,

Is it possible to query the bwa index for all locations of a single kmer using prophex?

**Comment by karel-brinda** [comment]
No, ProphEx doesn't output this. But you can use bwa fastmap (see eg https://github.com/karel-brinda/simplitigs-supplementary/blob/43e09b2052e64eb92fdbea8ee669becb1489ed0e/experiments/multiple_pangenomes_bwa/Snakefile#L54 for how it can be done).

**Comment by Adamtaranto** [comment]
I was wondering if prophex had an internal method for querying a bwa index that I might be able to re-purpose. 

Thanks for the link I'll check out fastmap.
